tests/selenium_ex_tests/test_selenium_ex1.py

```diff
@@ -1,8 +1,43 @@
 from selenium import webdriver
 from selenium.webdriver.common.by import By
+
+
 driver = webdriver.Chrome()
 driver.get('http://www.google.com')
+
+
+try:
+    driver.find_element_by_id('gbw')
```

Comment:
Being your first selenium test, you should comment each step in order to reflect what you are trying to test in page. Classes from google.com are not specific, so it's hard to find out what you are looking at within the page.